torchvision/csrc/io/image/cpu/decode_gif.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <torch/types.h>
+
+namespace vision {
+namespace image {
+
+C10_EXPORT torch::Tensor decode_gif(const torch::Tensor& encoded_data);
```

Comment:
Add a comment about input tensor being contiguous as a requirement for this function?